habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -409,9 +409,12 @@ def update_base(self, fix_leg=False):
         end_pos = self._sim.step_filter(
             rigid_state.translation, target_rigid_state.translation
         )
-        # Offset the base if the base height is different between end_pos
-        # and the current state
-        if end_pos[1] != rigid_state.translation[1]:
+
+        # try_step may fail, in which case it simply returns the start argument
+        did_try_step_fail = end_pos == rigid_state.translation
```

Comment:
I guess this is fine. But going forward, let's keep our PRs into the HITL codebase separate from our PRs into Habitat-lab/baselines.